brownie/buyback.py

```diff
@@ -45,6 +44,11 @@ def build_buyback_tx(max_dollars=5000, max_slippage=2.0):
       Percentage of slippage from current prices to allow.
     """
     buyback = BUYBACK_CONTRACT
+    treasuryBps = BUYBACK_CONTRACT.treasuryBps()
+    ousd_for_treasury = max_dollars * treasuryBps / 1e4
```

Comment:
I've been doing this wrong all this time, but 1e4 / 1e18 return floats. For the precise int's we should use 10**4 or 10**18 in python.

<img width="105" alt="image" src="https://github.com/OriginProtocol/origin-dollar/assets/837/8defae3e-edaa-48a6-959b-d19b6d8e4620">

Reply:
Oh wow me too -.-. Changing all to ints